README.md

```diff
@@ -24,6 +24,7 @@ Just invoke *logme* in your Fastfile action. Logme will return the commit messag
 | `to_revision` | End revision commit | false |
 | `from_revision` | From revision commit | false |
 | `message_regexp_filters` | Regexp filters like. ie '^MDM-\|^CTS:' | true |
+| `message_regexp_exclude_filters` | Regexp filters like. ie '^MDM-TST\|^CTS:UI' | true |
```

Comment:
@p4checo 
It doesn't matter who runs first (I think!)
["MDM-cenas", "NOJIRA - UI", "MDM-coisas", "batatas"].select{|c| c[/^MDM-|^NOJIRA/]!=nil}.select{|c| c[/^NOJIRA.*UI/] == nil}
 => ["MDM-cenas", "MDM-coisas"]
["MDM-cenas", "NOJIRA - UI", "MDM-coisas", "batatas"].select{|c| c[/^NOJIRA.*UI/] == nil}.select{|c| c[/^MDM-|^NOJIRA/]!=nil}

Having in the same regexp, couldn't be very complex?